pkgs/applications/misc/yambar-hyprland-wses/default.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "yambar-hyprland-wses";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "jonhoo";
+    repo = pname;
```

```suggestion
    repo = "yambar-hyprland-wses";
```
Since `src.repo` and `pname` are semantically different, `repo = pname` is not recommended.